packages/init/templates/tsconfig.json.hbs

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
so this really isn't something we can share via e.g. a library?

Reply:
I think you can. you'd just reference the tsconfig in the `node_modules` folder